keras_cv/visualization/plot_image_gallery.py

```diff
@@ -25,14 +26,16 @@
 def plot_image_gallery(
     images,
     value_range,
-    rows=3,
-    cols=3,
     scale=2,
+    rows=None,
+    cols=None,
+    batch_size=8,
     path=None,
     show=None,
     transparent=True,
     dpi=60,
     legend_handles=None,
+    image_key="image",
```

Comment:
Let's keep this out of the visualization API, as it's not super relevant to the visualization component.

We have a pattern in these visualization tools of expecting only images in the `images` parameter, and we should keep it that way by assuming that the contents of the tf dataset are just image tensors (and documenting this assumption)

Reply:
Got it, removed